snapshots/devmapper/config.go

```diff
@@ -82,6 +85,13 @@ func (c *Config) parse() error {
 		return errors.Wrapf(err, "failed to parse base image size: '%s'", c.BaseImageSize)
 	}
 
+	switch c.FileSystemType {
+	case string(fsTypeExt4), string(fsTypeXfs):
+	case "":
+		c.FileSystemType = string(fsTypeExt4)
+	default:
+		return fmt.Errorf("unsupported Filesystem Type: %q", c.FileSystemType)
```

Comment:
Let's move this to `Validate()` ? (`if==""` check can remain in parse).

Reply:
Sounds good. I moved it to validate and kept if = "" part in parse. Thanks